examples/rendezvous/src/bin/register.rs

```diff
@@ -131,13 +131,20 @@ impl From<ping::Event> for MyEvent {
     }
 }
 
+impl From<Void> for MyEvent {
+    fn from(event: Void) -> Self {
+        void::unreachable(event)
+    }
+}
+
 #[derive(NetworkBehaviour)]
 #[behaviour(
     out_event = "MyEvent",
```

Comment:
In a recent version of the `NetworkBehaviour` derive, we add the ability to auto-generate this enum. I think our examples should use that as we consider it the recommended way of using the derive. Can you adapt the examples please? The only thing that will needed changing is that the generated enum will be called `MyBehaviourEvent`.